v3/ecl/sss/v2/users/urls.go

```diff
@@ -0,0 +1,23 @@
+package users
+
+import "github.com/nttcom/eclcloud/v3"
+
+func listURL(client *eclcloud.ServiceClient) string {
+	return client.ServiceURL("users")
+}
+
+func getURL(client *eclcloud.ServiceClient, tenantID string) string {
```

Comment:
```suggestion
func getURL(client *eclcloud.ServiceClient, userID string) string {
```
同一ファイルの他箇所にも適用お願いします。